src/clients/memberclient.cpp

```diff
@@ -291,6 +325,12 @@ int main(int argc, char** argv)
       "--nodes_to_add", nodes_file, "The file containing the nodes to be added")
     ->required(true);
 
+  std::string new_code_id;
+  auto add_code = app.add_subcommand("add_code", "Support executing new code");
+  add_code
+    ->add_option("--new_code_id", new_code_id, "The path of the new code id")
```

Comment:
Is this a path for a string? We could also give an example or a hint about the format.